notebooks/parsing_mbox.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>I think this wants to open an mbox file stored locally. The first argument should be a path name, not the body of a file. Saving the object to "data" first might help. See this notebook for another way to read the mbox files https://github.com/aicoe-aiops/mailing-list-analysis-toolkit/blob/master/notebooks/stage/raw_to_text.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>